### PR TITLE
feat: replaced FlexLayout with Tailwind in demo, elements and integration

### PR DESCRIPTION
**Pull request by ronnymikalsen**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/NationalLibraryOfNorway/ngx-mime/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[X] Feature
[] Code style update (formatting, local variables)
[] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
Replaced FlexLayout with Tailwind 

## Does this PR introduce a breaking change?

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: apps/demo/src/app/core/sidenav/sidenav.component.html [hunk 1]
```diff
@@ -27,22 +27,27 @@ <h1>Manifest</h1>
...
+          <img
+            src="assets/logo-iiif.png"
+            alt="IIIF Drag-n-drop"
+            class="logo-iiif"
+          /><div>DnD</div>
+        </div>
       </a>
       <a
         mat-list-item
+        class="flex flex-row"
```
Comment (raymondkarstensen) [inline apps/demo/src/app/core/sidenav/sidenav.component.html:40]: I don't think this is needed? Cannot see any effect by using it, and none of the other anchor elements has it.

File: apps/demo/tailwind.config.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const { createGlobPatternsForDependencies } = require('@nrwl/angular/tailwind');
+const { join } = require('path');
+
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    join(__dirname, 'src/**/!(*.stories|*.spec).{ts,html}'),
```
Comment (raymondkarstensen) [inline apps/demo/tailwind.config.js:7]: I guess this is generated by nrwl? But do we need to exclude `.stories` files we don't have?

File: apps/integration/tailwind.config.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+const { createGlobPatternsForDependencies } = require('@nrwl/angular/tailwind');
+const { join } = require('path');
+
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    join(__dirname, 'src/**/!(*.stories|*.spec).{ts,html}'),
```
Comment (raymondkarstensen) [inline apps/integration/tailwind.config.js:7]: I guess this is generated by nrwl? But do we need to exclude .stories files we don't have?

**Review by raymondkarstensen — CHANGES_REQUESTED** [review]

**Comment by raymondkarstensen** [comment]
ViewerModule in integration app still imports FlexLayoutModule

**Comment by raymondkarstensen** [comment]
Spec file for ViewerComponent still imports FlexLayoutModule

**Comment by ronnymikalsen** [comment]
> ViewerModule in integration app still imports FlexLayoutModule

Fixed

**Comment by ronnymikalsen** [comment]
> Spec file for ViewerComponent in integration app still imports FlexLayoutModule

Fixed

File: apps/demo/src/app/core/navbar/navbar.component.html
[the remark this answers is not in the document]
Comment (ronnymikalsen) [reply apps/demo/src/app/core/navbar/navbar.component.html:1]: Using your suggestion :-)

**Comment by ronnymikalsen** [comment]
> AppComponent in demo app uses `MediaChange` and `MediaObserver` from flex-layout.

Fixed

File: apps/demo/src/app/core/sidenav/sidenav.component.html [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <h1>Manifest</h1>
-<div class="api-format" fxLayout="column" fxLayoutAlign="space-around start">
+<div class="api-format columns-1">
```
[the remark this answers is not in the document]
Comment (ronnymikalsen) [reply apps/demo/src/app/core/sidenav/sidenav.component.html]: Fixed

File: apps/demo/src/app/core/sidenav/sidenav.component.html (hunk 1 above)
Comment (ronnymikalsen) [reply apps/demo/src/app/core/sidenav/sidenav.component.html]: Fixed

File: apps/demo/tailwind.config.js (hunk 2 above)
Comment (ronnymikalsen) [reply apps/demo/tailwind.config.js]: Fixed by removing storybook config

File: apps/integration/src/app/viewer/viewer.component.html [hunk 5]
```diff
@@ -1,4 +1,4 @@
-<div class="viewer-container" fxLayout="column" fxFill>
+<div class="viewer-container w-full h-full flex flex-col">
   <div *ngFor="let manifestUri of manifestUris" [fxFlex]="viewerHeight">
```
[the remark this answers is not in the document]
Comment (ronnymikalsen) [reply apps/integration/src/app/viewer/viewer.component.html]: Fixed

File: apps/integration/tailwind.config.js (hunk 3 above)
Comment (ronnymikalsen) [reply apps/integration/tailwind.config.js]: Fixed by removing storybook config

File: apps/demo/src/app/core/navbar/navbar.component.html [hunk 6]
```diff
@@ -1,27 +1,24 @@
...
-          autocomplete="off"
-          [(ngModel)]="manifestUri"
-          name="manifestUri"
-        />
-        <button type="submit" mat-raised-button color="accent">VIEW</button>
-      </div>
+<mat-toolbar color="primary">
+  <button mat-icon-button (click)="toggle()" aria-label="Menu">
+    <mat-icon class="mat-24">menu</mat-icon>
+  </button>
```
Comment (jimarthurnilsen) [inline apps/demo/src/app/core/navbar/navbar.component.html:4]: Can we add some gap between the menubutton and the inputfield?

**Review by jimarthurnilsen — APPROVED** [review]

**Pull request merged**